index.html

```diff
@@ -404,7 +411,7 @@
 
     <section id="followers">
       <h2>Followers Collection</h2>
-      <p>Every <a href="#actors">actor</a> MUST have a followers collection.
+      <p>Every <a href="#actors">actor</a> MAY have a followers collection.
```

Comment:
Changing the MUST to a MAY (re an actor's `followers` collection) per previous discussion in issue https://github.com/w3c-social/activitypub/issues/69